wordpress/wp-content/themes/_bbr/page-books.php

```diff
@@ -1,17 +1,50 @@
 <?php
 /*
-Template Name: Books list
+Template Name: Book List
 */
 
-get_header();
-?>
+get_header(); ?>
 
-<div id="books" class="content full"></div>
-
-<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/3.8.0/lodash.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-<script src="<?= get_template_directory_uri() ?>/js/react-0.13.2.min.js"></script>
-<script src="<?= get_template_directory_uri() ?>/js/ReactRouter.min.js"></script>
-<script src="<?= get_template_directory_uri() ?>/js/books.js"></script>
+<div class="container">
+  <div class="row">
+    <div class="col-md-3 sidebar right-sidebar">
+      <div class="widget sidebar-widget box-style1">
+        <!-- TODO: figure out what is supposed to go over here, maybe nothing? -->
+        <h3 class="widget-title">Top categories</h3>
+        <ul class="top-categories-list">
+        </ul>
+      </div>
+      <div class="widget sidebar-widget box-style1">
+        <h3 class="widget-title">Categories</h3>
+        <ul class="categories-list">
+          <!-- TODO: not sure what this is for either -->
+        </ul>
+      </div>
+    </div>
+    <div class="col-md-9">
+      <ul class="sort-destination isotope exhibitions-grid" data-sort-id="grid">
+        <!-- TODO: loop through books with the following list items -->
+        <li class="col-md-4 col-sm-4 grid-item format-standard accrue-homestead">
+          <div style="width:100%;height:240px;background:url(http://overnight-website.s3.amazonaws.com/wp-uploads/CelestesHarlemRenaissance.jpg) center / cover;" />
+          <div class="grid-item-content">
+            <h3 style="height:81px;overflow:hidden;">book.title</h3>
+            <!-- TODO: not sure what this overload thing is, maybe I meant Overdrive? -->
+            <div class="meta-data grid-item-meta"><i class="fa fa-clock-o"></i> Available at Overload</div>
```

Comment:
I think we can take this out, since Overdrive integration is not being implemented.